contributing/notebook_template/notebook_template.ipynb

```diff
@@ -29,21 +29,19 @@
     }
    },
    "source": [
+    "## Table of Contents\n",
+    "* [Introduction](#Introduction)\n",
+    "* [Main Content](#Main-Content-(Rename))\n",
+    "    * [Loading Data](#Loading-Data-(Rename))\n",
```

Comment:
In case you take the suggestion below, this would be:
`[Loading Data](#Loading-data-(Raname,-e.g.-Querying-for-TESS-Data))\n`